nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/PeakNavigator.java

```diff
@@ -207,7 +208,20 @@ void initPeakNavigator(ToolBar toolBar, PeakNavigator parentNavigator, MenuButto
         };
 
         ProjectBase.getActive().addPeakListListener(mapChangeListener);
-
+        // The different control items end up with different heights based on font and icon size,
+        // set all the items to use the same height
+        this.navigatorToolBar.heightProperty().addListener((observable, oldValue, newValue) -> {
+            // don't adjust the height of the close button which is always at index 0
+            List<Node> navToolBarItems = this.navigatorToolBar.getItems().subList(1, this.navigatorToolBar.getItems().size());
+            Optional<Double> height = navToolBarItems.stream().map(node -> node.prefHeight(Region.USE_COMPUTED_SIZE)).max(Double::compare);
+            if (height.isPresent()) {
+                for (Node node : navToolBarItems) {
+                    if (node instanceof Control) {
+                        ((Control) node).setMaxHeight(height.get());
+                    }
+                }
+            }
+        });
```

Comment:
Can you export these to a utility class? Since there's a lot of code repetition you could just pass the toolbar that you are looking to add this listener to.

Reply:
Moved this code to a method in GUIUtils